src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String helloString;
```

Comment:
I would say the inclusion of the data type (`String` in this case) in the variable name can be considered redundant, as we already know that it is string, however it is rather point to discuss than strict requirement